unix-domain-socket/src/test/scala/docs/scaladsl/UnixDomainSocketSpec.scala

```diff
@@ -72,19 +72,15 @@ class UnixDomainSocketSpec
 
       val sendBytes = ByteString("Hello")
 
-      val binding =
-        UnixDomainSocket().bindAndHandle(Flow[ByteString]
-                                           .delay(5.seconds)
-                                           .map(identity),
-                                         file)
+      val binding = UnixDomainSocket().bindAndHandle(Flow[ByteString].map(identity), file)
```

Comment:
Can you also please comment on why the existing tests are changing?

Reply:
I'll revert the code, there is no reason to change it.